suites/apis/centralizedAuth.js

```diff
@@ -0,0 +1,655 @@
+
+BeforeSuite(async (fence, users, indexd) => {
+  console.log('Removing test indexd records if they exist');
+  await indexd.do.deleteFileIndices(Object.values(new_gen3_records));
+  await indexd.do.deleteFileIndices(Object.values(new_abc_records));
+  await indexd.do.deleteFileIndices(Object.values(indexed_files));
+});
+
+AfterSuite(async (fence, indexd, users) => {
+  console.log('Removing indexd files used to test signed urls');
+  await indexd.do.deleteFileIndices(Object.values(indexed_files));
+});
+
+Before(async (fence, users, indexd) => {
+  // populate new GUIDs per test
+  const gen3FooBarFileGuid = uuid.v4().toString();
+  const gen3DeleteMe = uuid.v4().toString();
+  const abcFooBarFileGuid = uuid.v4().toString();
+  const abcDeleteMe = uuid.v4().toString();
+
+  new_gen3_records.fooBarFile.did = gen3FooBarFileGuid;
+  new_gen3_records.deleteMe.did = gen3DeleteMe;
+  new_abc_records.fooBarFile.did = abcFooBarFileGuid;
+  new_abc_records.deleteMe.did = abcDeleteMe;
+
+  console.log('Adding indexd files used to test signed urls');
+  const ok = await indexd.do.addFileIndices(Object.values(indexed_files));
+  chai.expect(
+    ok, 'unable to add files to indexd as part of centralizedAuth setup').to.be.true;
+});
+
+After(async (fence, users, indexd) => {
+  // Cleanup after each scenario
+  console.log('Removing test indexd records if they exist');
+  await indexd.do.deleteFileIndices(Object.values(new_gen3_records));
+  await indexd.do.deleteFileIndices(Object.values(new_abc_records));
+});
+
+/*
+   - Test that user without policies can't CRUD records in /gen3 or /abc
+*/
+Scenario('User without policies cannot CRUD indexd records in /gen3 or /abc @centralizedAuth',
+  async (fence, indexd, users, files) => {
+    return; // FIXME: skip test for now
+    // create
+    const gen3_create_success = await indexd.do.addFileIndices(
+        Object.values(new_gen3_records), users.user2.accessTokenHeader);
+    const abc_create_success = await indexd.do.addFileIndices(
+        Object.values(new_abc_records), users.user2.accessTokenHeader);
+
+    // read
+    const gen3_read_response = indexd.do.getFile(
+        new_gen3_records.fooBarFile, users.user2.accessTokenHeader)
+    const abc_read_response = indexd.do.getFile(
+        new_abc_records.fooBarFile, users.user2.accessTokenHeader)
+
+    // update
+    const filename_change = {
+      'file_name': 'test_filename'
+    }
+    const gen3_update_response = indexd.do.updateFile(
+        new_gen3_records.fooBarFile, filename_change, users.user2.accessTokenHeader)
+    const abc_update_response = indexd.do.updateFile(
+        new_abc_records.fooBarFile, filename_change, users.user2.accessTokenHeader)
+
+    // delete
+    const gen3_delete_response = indexd.do.deleteFile(
+        new_gen3_records.deleteMe, users.user2.accessTokenHeader)
+    const abc_delete_response = indexd.do.deleteFile(
+        new_abc_records.deleteMe, users.user2.accessTokenHeader)
+
+    // asserts for create
```

Comment:
for these tests i think it would be safe to do the asserts right after the tests

Reply:
:+1: